### PR TITLE
Convert min date of timestamp columns in SimpleSqlServerTransformer

### PR DESCRIPTION
**Pull request by FarBo**
This PR solves the problem explained in this issue #64

### CODE REVIEW

File: src/spetlr/transformers/simple_sql_transformer.py [hunk 1]
```diff
@@ -33,4 +37,18 @@ def process(self, df: DataFrame) -> DataFrame:
         ]
         df = df.select(col_choose)
 
+        # The sql server min date is 1753-01-01. If a timestamp column of the
```
Comment (LauJohansson) [inline src/spetlr/transformers/simple_sql_transformer.py:40]: Create a test of the logic. Could be added here: https://github.com/spetlr-org/spetlr/blob/main/tests/cluster/sql/test_simple_sql_etl.py

Comment (FarBo) [reply src/spetlr/transformers/simple_sql_transformer.py:40]: I couldn't find the test file for this module. Thanks for the hint
I will include the test there.

Comment (LauJohansson) [reply src/spetlr/transformers/simple_sql_transformer.py:40]: I will approve when you have created the tests.

**Review by LauJohansson — APPROVED** [review]
I approve. I believe you fix the test error by choosing a datetime before 1900, but still a time that the dt_utc can take as input

**Pull request merged**